index.js

```diff
@@ -60,6 +60,7 @@ export default postcss.plugin('postcss-lab-function', opts => {
 							)
 
 							children[0].value = String(rgbValues[0]);
+							children[0].unit = '';
```

Comment:
this feels kinda hacky, but seemed easier then chasing down where the child get's extended and adopts the percentage.